app/models/character_type.rb

```diff
@@ -1,7 +1,9 @@
 class CharacterType < ApplicationRecord
-  belongs_to :user
+  belongs_to :user, optional: true, inverse_of: :character_types
```

Comment:
@StefanGajic why is this optional?

Reply:
Again I was looking at some examples, and I forgot to remove it later. I understand we don't want user to be optional